client/components/grid.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <div class="grid">
+    <slot></slot>
+  </div>
+</template>
+
+<style lang="stylus">
+.grid {
+  display: flex;
```

Comment:
Can you use grid instead of flex?

Reply:
I'll see if it still works using grid

Reply:
I was having trouble getting grid to work as it wasn't wrapping the row to next line in smaller viewports.